src/lsp.rs

```diff
@@ -88,24 +119,30 @@ impl<TLocalLinter: LocalLinter> Backend<TLocalLinter> {
                 per_file_state.supported_language_language,
             )
         };
-        let per_config_context = self.per_config_context.read().await;
+        let per_config_context = self.per_config_context.get();
```

Comment:
Per previous PR comment it turned out I could use `OnceLock` for this because it does support a "check if the value is populated" API